quality_control_oca/models/qc_inspection.py

```diff
@@ -16,8 +16,10 @@ class QcInspection(models.Model):
 
     @api.depends("inspection_lines", "inspection_lines.success")
     def _compute_success(self):
-        for i in self:
-            i.success = all([x.success for x in i.inspection_lines])
+        for inspection in self:
+            lines = inspection.inspection_lines
+            inspection.success = all([x.success for x in lines])
+            inspection.count_success = len(lines.filtered(lambda x: x.success is True))
```

Comment:
I think this may be a better approach
```suggestion
            inspection.success = all([x.success for x in lines])
            inspection.count_success = len(inspection.success)
```

Reply:
Hi @Christian-RB , in this case, the lines that are successful are counted to display in the kanban view of the inspections.